addon-resizer/nanny/kubernetes_client_test.go

```diff
@@ -273,6 +273,63 @@ func TestGetNodeCountFromDecoder(t *testing.T) {
 			finalResult: fmt.Errorf("oops"),
 			expectErr:   fmt.Errorf("decoding error: oops"),
 		},
+		{
+			name:        "fallbacks on error preferred on preferred metric",
```

Comment:
I can't parse this test case's name

Reply:
Changed to "falls back on error in preferred metric" (the test name I tried to write)